pkg/universe.dagger.io/terraform/test/test.cue

```diff
@@ -0,0 +1,35 @@
+package terraform
+
+import (
+	"dagger.io/dagger"
+
+	"universe.dagger.io/terraform"
+)
+
+let tfversion = "1.1.7"
+
+dagger.#Plan & {
+	client: filesystem: "./data": read: contents: dagger.#FS
```

Comment:
We should change our tests to use `core.#Source` instead of `client: filesystem`. If you feel like being on the vanguard, you can change it. But it's still mergeable 🙂

Reply:
Hey, @helderco Thanks. Do you have some more details/examples on this?

I'm also a bit limited over the next few days. Do you mind merging the current state (with the current limitations mentioned in the `// TODO` lines)? So maybe someone else can step in and continue to expand this. Otherwise, I can continue to work on it in the future.